home.html

```diff
@@ -143,7 +214,7 @@
     <div class="footer">
         <div class="footer__top">
             <div class="footer__logo">
-                <img src="/Assets/images/home-footer/logo_footer.svg" alt="Company logo" />
+                <img class:"" src="/Assets/images/home-footer/logo_footer.svg" alt="Company logo" />

```

Comment:
This is still here

Reply:
Line 217 is part of the footer section, i actually took care of that on the other pull request :)

Reply:
The only reason why i didnt change that  in this PR is because i wanted to avoid conflicts, if that make sense?